jekyll/_cci2/optimization-cookbook.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: classic-docs
+title: "CircleCI Optimizations Cookbook"
+short-title: "Optimizations Cookbook"
+description: "Starting point for Optimizations Cookbook"
+categories: [getting-started]
+order: 1
+---
+
+The *CircleCI Optimizations Cookbook* is a collection of individual use cases (referred to as "recipes") that provide you with detailed, step-by-step instructions on how to perform various optimization tasks using CircleCI resources. This guide, and it associated sections, will enable you to quickly and easily perform easily repeatable optimization tasks on the CircleCI platform.
+
+* TOC
+{:toc}
+
+## Introduction
+
+When work with the CircleCI platform, one of the most critical tasks you will want to complete is to optimize your jobs, builds, workflows, and other tasks to prevent performance degradation, latency, and concurrency. Optimizing your tasks is not simply a function of lessening the workload or number of tasks, but rather, reviewing your optimization and caching strategies to ensure you are meeting your business and technical requirements while also maintaining stability and reliability.
+
+CircleCI has developed several different approaches you may wish to use to better optimize these tasks while also leveraging the power of the CircleCI platform and its associated resources (e.g. orbs). The sections below describe various strategies and methodologies you can use to optimize your organization tasks.
+
+### Optimization Recipes
+
+The table below lists the different optimization tasks that can be performed using the CircleCI platform.
+
+Recipe | Description 
+------------|-----------
+Single-Threading Builds (Queueing) | This section describes how you can use a specific single-threading (queueing) orb to ensure proper job and build completion without concurrency (queueing subsequent builds or jobs).
+Caching |  This section describes how you can use different caching strategies in your implemenations to optimize your builds and workflows.
+Workflow Optimizations | This section describes how you can optimize your workflows.
+
+## Single-Threading Builds (Queuing)
```

Comment:
I'm not clear about the use-case for this.

I thought the queue orb was specifically about managing access to an external shared resource?

Reply:
+1, I'm also unsure the application of "thread" in this case makes sense. Given our system, I would consider parallel containers to be more like threads than parallel builds (for example). Referencing terms that have explicit meaning when applied ambiguously will leave it up to the reader to determine the meaning which can lead to misinterpretation.

Reply:
Running Jobs Sequentially?
Forcing Jobs to Run Sequentially?

Reply:
Re-titled section to "Running Jobs Sequentially" and made some content changes to address these wording issues.